Back End/InsiteTeamTask/Controllers/DataFilterController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using InsiteTeamTask.Models;
+using InsiteTeamTask.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InsiteTeamTask.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DataFilterController : ControllerBase
+    {
+        private readonly DataFilterService _dataFilterservice;
+
+        public DataFilterController(DataFilterService dataFilterService)
+        {
+            _dataFilterservice = dataFilterService;
+        }
+
+        [HttpGet("seasons")]
+        public ActionResult<IEnumerable<int>> GetAllSeasons()
```

Comment:
Ideally, you'd expect a controller action named "GetAllSeasons" to return all the Season objects, rather than just their IDs. It wasn't exactly necessary here, but if you continued this pattern it would mean you could use game names in the front end rather than having users pick IDs.